node/node_test.go

```diff
@@ -19,9 +19,6 @@ var MockServerAddr = ":7980"
 
 // cleanUpNode stops the node and checks if it is running
 func cleanUpNode(node Node, t *testing.T) {
-	defer func() {
-		node.Cancel()
-	}()
```

Comment:
this is unnecessary since the `node.Stop` method calls the `cancel` func that `node.Cancel` calls.